python_transport/wirepas_gateway/transport_service.py

```diff
@@ -73,6 +217,27 @@ def __init__(self, settings, logger=None, **kwargs):
 
         self.logger.info("Gateway started with id: %s", self.gw_id)
 
+        self.monitoringThread = None
```

Comment:
Could you avoid using camelCase for this variable please?

Reply:
Sure!!
Could we enforce it to avoid this kind of mistake?